NOTICE

```diff
@@ -1,5 +1,5 @@
-   Apache Metron
-   Copyright 2015-2020 The Apache Software Foundation
+   The Initial Developer of this project, which was copied from, derived from,
```

Comment:
I am not sure if what I did here was reasonable, but it was based off of [the ASF's example notice](https://infra.apache.org/licensing-howto.html#example-notice) and [this stackexchange answer](https://opensource.stackexchange.com/a/4440)

Reply:
I think to top should be Zeek-Kafka
with he new copyright.
Followed by this